apps/frontend/src/app/app.module.scss

```diff
@@ -1 +1,6 @@
 /* Your styles goes here. */
+
+body {
+    width: 1200px;
+    margin: 0 auto !important;
```

Comment:
We should also avoid using "!important": https://uxengineer.com/css-specificity-avoid-important-css/

Reply:
Aware of this. The design system seems to interfere with the margin, and I have yet to find the correct workaround. Will attempt to use the Panel component, and if centering is still an issue I will leave a comment to indicate that it should be worked out. For MVP purposes it should not become an issue as it uses css modules.

Reply:
Using Panel took care of this.